tests/test_models.py

```diff
@@ -28,18 +28,36 @@ def test_get_record(client):
         assert attr.asdict(rec_obj)['metadata'] == json_object['metadata']
 
 
-# def test_filtered_item_search(client):
-#     """Test filtered_item_search function."""
-#     item_links = client.filtered_item_search(key, string, query_type,
-#                                              selected_collections='')
-#     assert False
-#
-#
-# def test__pop_inst(client):
-#     rec_obj = client._pop_inst(class_type, rec_obj)
-#     assert False
-#
-#
-# def test__build_uuid_list(client):
-#     child_list = client._build_uuid_list(self, rec_obj, kwargs, children)
-#     assert False
+def test_filtered_item_search(client):
+    """Test filtered_item_search function."""
+    with requests_mock.Mocker() as m:
+        key = 'dc.title'
+        string = 'test'
+        query_type = 'contains'
+        selected_collections = ''
+        url = f'/rest/filtered-items?query_field[]={key}&query_op[]='
+        url += f'{query_type}&query_val[]={string}&collSel[]='
+        url += f'{selected_collections}&limit=200&offset='
```

Comment:
Here again I would start with the query string as a dictionary. Since you aren't passing to requests in this case, you can use https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode to create the query string. Long formatting strings with lots of variables should generally be avoided if possible. They are too prone to typos and difficult to debug.